### ISSUE TITLE
"Marker scan hit start of image data" should not be the WARN

### ISSUE THREAD

**Issue by niwics** [james-see/iptcinfo3#24]
When parsing JPEG files with no IPTC data, I suppose that the scanning message *Marker scan hit start of image data* should not be logged as warning, but as info. At least when I use the `force=True` flag in `IPTCInfo` constructor (intended for processing files without IPTC data).

Maybe I am missing something, but I don't see the possibility to determine whether my image has IPTC data or not without warnings.

**Comment by jamesacampbell** [comment]
@niwics That depends on what assumptions the package should have re: IPTC data present in images it is scanning. I feel like if force=True, it should just change it to INFO instead of WARNING, since you are basically overriding the traditional approach of getting data about the image being loaded.

**Comment by niwics** [comment]
I agree with that - when I use the flag `force=True`, I assume that no IPTC data are there so INFO log level is appropriate.

**Comment by james-see** [comment]
minor issue compared to supporting latest IPTC tags that still need work. closing for now unless sponsored.

**Comment by james-see** [comment]
When `force=True` is used in the IPTCInfo constructor, the "Marker scan hit start of image data" message is now logged at INFO level instead of WARNING.

This makes sense because when you use `force=True`, you're explicitly indicating that you expect files without IPTC data, so it shouldn't be treated as a warning condition.

Thank you for the suggestion!
